src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="im.tox.toktok">
-    <uses-sdk android:minSdkVersion="22"
```

Comment:
Why did you remove this? I'm getting warnings without it.

Reply:
The Gradle sync thing fails with it there:
```
ERROR: The minSdk version should not be declared in the android manifest file. You can move the version from the manifest to the defaultConfig in the build.gradle file.
Remove minSdkVersion and sync project
Affected Modules: toktok-android
```

Reply:
I added them to the BUILD.bazel, so now that should work again.